<launch>
  <arg name="velodyne_monitor_param_file" default="$(find-pkg-share autoware_velodyne_monitor)/config/VLP-16.param.yaml"/>
  <arg name="ip_address" default="192.168.1.201"/>

  <node pkg="autoware_velodyne_monitor" exec="autoware_velodyne_monitor_node" name="velodyne_monitor" output="log" respawn="true">
    <param from="$(var velodyne_monitor_param_file)"/>
    <param name="ip_address" value="$(var ip_address)"/>
  </node>
</launch>
